packages/xo-server/src/api/host.js

```diff
@@ -2,6 +2,57 @@ import { format } from 'json-rpc-peer'
 
 // ===================================================================
 
+export async function getSchedulerGranularity({ host }) {
+  try {
+    return await this.getXapi(host).getField(
+      'host',
+      host._xapiRef,
+      'sched_gran'
+    )
+  } catch (e) {
+    if (e.code === 'MESSAGE_METHOD_UNKNOWN') {
+      return null
+    }
+    throw e
+  }
+}
+
+getSchedulerGranularity.description = 'get the scheduler granularity of a host'
+
+getSchedulerGranularity.params = {
+  id: { type: 'string' },
+}
+
+getSchedulerGranularity.resolve = {
+  host: ['id', 'host', 'view'],
+}
+
+// ===================================================================
+
+export async function setSchedulerGranularity({ host, schedulerGranularity }) {
+  await this.getXapi(host).setField(
+    'host',
+    host._xapiRef,
+    'sched_grand',
```

Comment:
`sched_gran`?

Reply:
It's the name of the method
see https://github.com/vatesfr/xen-orchestra/issues/5291#issuecomment-699913298